data/json/items/melee/spears_and_polearms.json

```
@@ -601,6 +601,27 @@
     "qualities": [ [ "COOK", 1 ] ],
     "flags": [ "SPEAR", "REACH_ATTACK", "DURABLE_MELEE" ]
   },
+  {
+    "id": "pollaxe",
+    "type": "GENERIC",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": { "str": "pollaxe", "str_pl": "pollaxes" },
+    "description": "This is a polearm with an axe head that's smaller then a halberd, a small hammer on the back, and a dagger-like spike which is generally shorter then most polearm spikes on top. It's less heavier then most polearms and has a smaller head, making it slightly quicker.",
```

I'm not sure the comparisons to other polearms are all that helpful, and they seriously break up the flow of the sentence.
```suggestion
    "description": "This is a polearm with an axe head, a small hammer on the back, and a short dagger-like spike on top.  It's less heavy then most polearms and has a smaller head, making it slightly quicker.",
```

---

sorry, got a little bit carried away there i think. i love medival weapons so i kinda drifted off :)